foods.html

```diff
@@ -26,13 +27,13 @@ <h1>Create New Food</h1>
 <div class='add-meal-selections'>
   <form id='new-food-form'>
     <div class='label'>Food Name</div><br>
-    <input type="text" name="new-food-name" id="new-food-name" required
-      oninvalid="this.setCustomValidity('Please enter a food name.')"
-      oninput="setCustomValidity('')"/><br>
+    <input type="text" name="new-food-name" id="new-food-name" required/>
+    <p class="validation-message">*Please enter a food name.</p>
```

Comment:
So this doesn't rely on the html validation? Is it a manually triggered component? Cool! Might need CSSing.